firebase-config/src/test/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigTest.java

```diff
@@ -189,6 +205,22 @@ public void setUp() throws Exception {
                 mockFireperfGetHandler,
                 RemoteConfigComponent.getMetadataClient(context, APP_ID, FIREPERF_NAMESPACE));
 
+    personalizationFrc =
+        FirebaseApp.getInstance()
+            .get(RemoteConfigComponent.class)
+            .get(
+                firebaseApp,
+                RemoteConfigComponent.DEFAULT_NAMESPACE,
```

Comment:
Do we need a different instance of RC for personalization? Using the default namespace will get the same RC instance as `frc` above.

Reply:
We need it to use a real `ConfigGetParameterHandler` so we can test that it's doing things correctly.